portal-api/src/v1/users/validation/rules/select-at-least-one-bank.js

```diff
@@ -0,0 +1,37 @@
+const { ADMIN } = require('../../../roles/roles');
+
+const CONSTANT = require('../../../../constants');
+/**
+ * Validates that if the bank is present it is not empty
+ * @param {Object} user the existing user
+ * @param {Object} change the changes to make
+ * @returns {Array} either an empty array or an array containing an error object
+ */
+const selectAtLeastOneBank = (user, change) => {
+  if (!change.bank || change.bank.length === 0 || (change.bank.length === 1 && change.bank[0] === null)) {
```

Comment:
This condition needs to be simplified please.

Reply:
Simplified condition and added test cases using suggested pattern.